### PR TITLE
fix/travis: Rust nigtly version is now set in the Clippy script

### PR DESCRIPTION
**Pull request by afck**

### CODE REVIEW

**Comment by maidsafe-highfive** [comment]
r? @dirvine

(maidsafe_highfive has picked a reviewer for you, use r? to override)

File: Cargo.toml [hunk 1]
```diff
@@ -11,6 +11,4 @@ version = "0.5.0"
 
 [dependencies]
 clippy = {version = "=0.0.99", optional = true}
-
-[dev-dependencies]
```
Comment (Fraser999) [inline Cargo.toml]: It's still a dev dependency though, isn't it?

File: src/lib.rs [hunk 2]
```diff
@@ -66,9 +66,6 @@
 #![cfg_attr(feature="clippy", deny(clippy))]
 #![cfg_attr(feature="clippy", allow(use_debug))]
 
-#[cfg(test)]
```
Comment (Fraser999) [inline src/lib.rs]: I thought the policy was to do all crate imports here for all modules, even test-only ones?

File: Cargo.toml (hunk 1 above)
Comment (afck) [inline Cargo.toml]: Yes: I added a comment why that doesn't work at the moment.

**Pull request merged**